pallets/randomness/README.md

```diff
@@ -0,0 +1,52 @@
+# Randomness
+
+This pallet provides access to randomness using as source the **BABE epoch randomness**, which is produced by the relay chain per relay chain epoch. It is based on **all the VRF produced** by the relay chain validators **during** a complete **epoch**.(~600 blocks on Kusama, ~2400 blocks on Polkadot). At the beginning of a new Epoch, those VRFs are **mixed together** and **hashed** in order to produce a **pseudo-random word**.
+
+## Babe Epoch Randomness
```

Comment:
I really like the explanations in this file, easy to understand 😄 .

Reply:
Thanks! 😃